analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.Classifier.cs

```diff
@@ -0,0 +1,203 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.UnitTest.Rules;
+
+public partial class TokenTypeAnalyzerTest
+{
+    [TestMethod]
+    public void ClassClassifications() =>
+        ClassifierTestHarness.AssertTokenTypes("""
+            [k:using] [u:System];
+            [k:public] [k:class] [t:Test]
+            {
+                [c:// SomeComment]
+                [k:public] [t:Test]() { }
+
+                [c:/// <summary>
+                /// A Prop
+                /// </summary>
+            ]   [k:int] [u:Prop] { [k:get]; }
+                [k:void] [u:Method]<[t:T]>([t:T] [u:t]) [k:where] [t:T]: [k:class], [t:IComparable], [u:System].[u:Collections].[t:IComparer]
+                {
+                    [k:var] [u:i] = [n:1];
+                    var s = [s:"Hello"];
+                    [t:T] [u:local] = [k:default];
+                }
+            [u:}]
+            """);
+
+    [TestMethod]
+    public void IndentifierToken_QueryComprehensions() =>
```

Comment:
Typo, same for the other calls, `Indentifier` -> `Identifier`
```suggestion
    public void IdentifierToken_QueryComprehensions() =>
```